CryptoTechReminderSystem/Gateway/HarvestGateway.cs

```diff
@@ -104,16 +117,30 @@ private JObject RetrieveFromEndPoint(string address)
 
         private JObject RetrieveWithPagination(string address)
         {
-            var apiResponse = RetrieveFromEndPoint($"{address}&page=1");
+            var apiResponse = RetrieveFromEndPoint(AppendURLWithPageNumber(address, 1));
+            
+            if (apiResponse["total_pages"] == null) {
+                Console.WriteLine(apiResponse);
+            }
+
             var totalPages = (int) apiResponse["total_pages"];
```

Comment:
This is a minor, personal stylistic preference: I would generally prefer an early `return apiResponse` here if `totalPages == 1`.
This is so that I don't have to infer why the next lines of code start looping at index 2 - the early return would already have given me that hint.
I wouldn't worry to change this though, I just thought it would be interesting to mention.

Reply:
I 100% agree!  This is a Martin Fowler guard clause, have stuck it in.  Thanks :D